docs/site_building/manage_page_display/embedded_formatter.rst

```diff
@@ -0,0 +1,49 @@
+Embedded Entity Formatter for Linker Fields
+============================================
+
+This formatter would be available to all Linker Fields in Tripal Chado. It would allow the admin to configure the view mode they want to use and it would embed a rendered version of the linked entity using that entity_id and view mode.
+
```

Comment:
```suggestion
This formatter is available to all Linker Fields in Tripal Chado. It allows the admin to configure the view mode they want to use and it will embed a rendered version of the linked entity using that entity_id and view mode.

```
I think it sounds better this way